bootstrap.env

```
@@ -1,4 +1,4 @@
-export UNIQUE_TEST_ID="$(uuidgen | tr "[:upper:]" "[:lower:]" | head -c 10)"
+export UNIQUE_TEST_ID="$(cat UNIQUE_TEST_ID 2>/dev/null || uuidgen | tr "[:upper:]" "[:lower:]" | head -c 10|tee UNIQUE_TEST_ID)"
```

I know its my fail but all the other pipes have spaces around them.
```suggestion
export UNIQUE_TEST_ID="$(cat UNIQUE_TEST_ID 2>/dev/null || uuidgen | tr "[:upper:]" "[:lower:]" | head -c 10 | tee UNIQUE_TEST_ID)"
```

---

No longer relevant because I removed this logic. Thank you though <3